src/chain/chain.rs

```diff
@@ -937,12 +937,7 @@ impl Chain {
     }
 
     /// Finds the `count` names closest to `name` in the whole routing table.
-    fn closest_known_names(
-        &self,
-        name: &XorName,
-        count: usize,
-        connected_peers: &[&XorName],
-    ) -> Vec<XorName> {
+    fn closest_known_names(&self, name: &XorName, count: usize) -> Vec<XorName> {
```

Comment:
If `count` is less than minimum section size, this will always be within a single section, anyway. We could revisit this, but I'm not sure if this PR is the right moment.